ban/http/api.py

```diff
@@ -574,16 +667,33 @@ def get_collection(self):
         """Get database diffs.
 
         parameters:
-        - name: increment
-          in: query
-          description: The minimal increment value to retrieve
-          type: integer
-          required: false
+            - name: increment
+              in: query
+              type: integer
+              required: false
+              description: The minimal increment value to retrieve
         responses:
-          200:
-            description: A list of diff objects
-            schema:
-              $ref: '#/definitions/Diff'
+            200:
+                description: A list of diff objects
+                schema:
+                    type: object
+                    properties:
+                        total:
+                            name: total
+                            type: integer
+                            description: total resources available
+                        collection:
+                            name: collection
+                            type: array
+                            items:
+                                $ref: '#/definitions/Diff'
+            400:
```

Comment:
I think we can just point to the generic 400 reference here, the detail will be in the response body.